src/main.c

```diff
@@ -69,6 +69,7 @@ static struct subcmd commands[] = {
 	{ "clean", "Clean cached files", clean_main },
 	{ "hashdump", "Dump the HMAC hash of a file", hashdump_main },
 	{ "verify", "NOTE: this command has been superseded, please use \"swupd diagnose\" instead", diagnose_main },
+	{ "3rd-party", "Indicate Swupd to use user's third party repo", third_party_main },
```

Comment:
And use the ifdef here too. I forgot to mention that in my first review

Reply:
Ahh yes.. will do

Reply:
Makes sense will change in next push